test/LegitimatePhygitalNFTv3.ts

```diff
@@ -463,7 +492,7 @@ describe('LegitimatePhygitalNFTv3', () => {
 
         const owner = await lgtNFT.ownerOf(tokenId1)
         expect(owner).to.eq(addr2.address)
-        const tokenLock = await lgtNFT.getTokenLock(tokenId1)
```

Comment:
can we leave the `getTokenLock` tests in as well? we need these to work with our existing system.

Reply:
i wrote an additional test to make sure that locked and getTokenLock returns the same result.